### DISCUSSION TITLE
DAI 2024-2025 - Check out my GitHub profile! - Thomas Stäheli

### DISCUSSION THREAD

**Discussion by thomasstaheli** [heig-vd-dai-course/heig-vd-dai-course#252] [Show and tell]
[https://github.com/thomasstaheli/thomasstaheli](url)

**Comment by ludelafo** [comment]
Nice profile pic, I like penguins myself. :)

How is the transition from electronics to computer sciences?

**Reply by thomasstaheli** [reply]
Thank you ! 
I want to learn more about embedded computing, because I think this is the most interesting part. :)

**Reply by ludelafo** [reply]
That's cool! Hope you'll find what you're looking for at HEIG-VD!
